projects/github-actions/repo-gardening/src/tasks/triage-issues/index.js

```diff
@@ -152,9 +152,9 @@ async function triageIssues( payload, octokit ) {
 		const issueLabels = await aiLabeling( payload, octokit );
 
 		// At this point, if we still miss a [Type] label, a [Feature] label, or a [Pri] label, ask the author to add it.
-		const requiredLabelTypes = [ '[Type]', '[Feature', '[Pri]' ];
+		const requiredLabelTypes = [ /^\[Type\]/, /^\[Pri/, /^\[.*?Feature/ ];
```

Comment:
Did you intend to leave off the `\]` from `[Pri]`?

Also, it might be good to avoid potentially matching something like `[Type] Feature Request`.

```suggestion
		const requiredLabelTypes = [ /^\[Type\]/, /^\[Pri\]/, /^\[[^\]]*Feature/ ];
```

Reply:
> Did you intend to leave off the \] from [Pri]?

I did, yes, I figured it'd be useful so we can catch `[Priority]` for repos that don't use the shorthand.

> it might be good to avoid potentially matching something like [Type] Feature Request.

Good point. Done in f0313ebd181bd3723668e2f6cf3b102aa51745b5